### PR TITLE
fix: typo resolved in the api2 package label description

### LINKED ISSUE #820: Typo in period defaults setting
Change: "How many redundant quantifications are assigned to each praise?"
To: "How many quantifiers are assigned to each praise?"

image

### PR DESCRIPTION
**Pull request by maniyuvan**
Resolves #820 

@kristoferlund  Updated "How many redundant quantifications are assigned to each praise? to: "How many quantifiers are assigned to each praise?" in the **api2** package.

### CODE REVIEW

**Comment by maniyuvan** [comment]
@kristoferlund hope, this is what you expected

**Comment by kristoferlund** [comment]
> @kristoferlund hope, this is what you expected

Unfortunately no. :) 

All migrations need to be there still, renaming 01 to 32 wont work. What happens if migration 02 need the settings to be there and you have deleted that migration? For someone setting up the system for the first time all migrations will run, in numerical order. The migration you need to write, number 31, should only change the label in question, nothing else.

**Comment by maniyuvan** [comment]
So no need to remove the old file, instead I need to create a file with 31_xx ?

**Comment by maniyuvan** [comment]
@kristoferlund  I've update the PR as you sugguested.

File: packages/api2/src/database/migrations/31_settings_add_label_description.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import { SettingModel } from '../schemas/settings/00_settings.schema';
+
+const settings = [
```
Comment (kristoferlund) [inline packages/api2/src/database/migrations/31_settings_add_label_description.ts:3]: The migration should only affect `PRAISE_QUANTIFIERS_PER_PRAISE_RECEIVER`. Now you include a whole list of settings that are not needed. Remove everything but `PRAISE_QUANTIFIERS_PER_PRAISE_RECEIVER`.

**Review by kristoferlund — CHANGES_REQUESTED** [review]

File: packages/api2/src/database/migrations/31_settings_add_label_description.ts (hunk 1 above)
Comment (maniyuvan) [reply packages/api2/src/database/migrations/31_settings_add_label_description.ts:3]: Updated the PR, please check now

**Pull request merged**